.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -25,32 +25,23 @@ assignees: ''
  If you know how to use git blame, that is the easiest way, otherwise, here is a rough guide of **who to tag**.
  Please tag fewer than 3 people.
 
- albert, bert, GPT2, XLM: @LysandreJik
- tokenizers: @mfuntowicz
- Trainer: @sgugger
- Speed and Memory Benchmarks: @patrickvonplaten
- Model Cards: @julien-c
- TextGeneration: @TevenLeScao
+ albert, bert, gpt2, xlm: @LysandreJik
+ bart, blenderbot, t5, speed and memory benchmarks, pegasus, longformer/reformer, marian, mbart: @patrickvonplaten
+ deepspeed, fsmt: @stas00
+ documentation, trainer: @sgugger
```

Comment:
I don't think it's helpful to mix different thins here: I liked having all models together before for instance, then other parts. So I'd rather reorganize by:
- models
- library components
- documentation
- examples

Reply:
That works for me. Let's agree with others that this what everybody likes and I will adjust

Let me know if this works and if you want the cases to be different - they are inconsistent now.

Something like?

```
Models:
 albert, bert, xlm: @LysandreJik
 blenderbot, bart, marian, pegasus, encoderdecoder, longformer, reformer, t5: @patrickvonplaten
 fsmt: @stas00
 gpt2: @LysandreJik, @patrickvonplaten
 rag: @patrickvonplaten, @lhoestq
 tensorflow: @jplu
 transfoxl, xlnet: @patrickvonplaten, @TevenLeScao
 tokenizers: @mfuntowicz

Library:
 benchmarks: @patrickvonplaten
 deepspeed: @stas00
 text generationt: @patrickvonplaten, @TevenLeScao
 trainer, @sgugger

Documentation:
 documentation: @sgugger
 model Cards: @julien-c

HF projects: 
 rust tokenizers: [different repo](https://github.com/huggingface/tokenizers)
 nlp datasets: [different repo](https://github.com/huggingface/nlp)

Examples:
 examples/bert-loses-patience: @JetRunner
 examples/distillation: @VictorSanh
 examples/seq2seq: @patil-suraj
 examples/token-classification: @stefan-it
```

Reply:
One more thing - the 2 sets (2 files) aren't the same. Should they be the same? And which one is the master then? I guess I can just merge the two inputs to get all the entry points.

Reply:
That's better! Now digging into the specifics I think (cc @LysandreJik and @patrickvonplaten ):
- under model you can add me for funnel
- for tokenizers we should remove poor Morgan. Do we put Anthony instead?
- The examples have changed. I think we should have a "maintained examples" line where you can put me and maybe @patil-suraj (if you don't mind starting helping maintaining all the new scripts?) and then two lines for the research projects mentioned (bert-loses-patience, distillation)

Reply:
I'm down for "maintained examples" @sgugger  :), also you could put me alongside Patrick for T5, bart, blender, marian, enc-dec

Reply:
OK, let's sort out the PR template and then sync it with Issue template.

I applied the new structure - please make direct suggestions to the code for any desired changes. I applied most of your suggestions except Morgan/Anthony

Reply:
Let's also remove the model cards and I think we can also remove @TevenLeScao  from everything (or do you still want to be pinged @TevenLeScao ?)

Reply:
Just talked offline with @n1t0 and he is okay to replace Morgan for the tokenizers (Morgan is working on lots of other stuff at the moment, so doesn't need those notifications ;-) )
Agreed with the new structure @stas00 and all the changes suggested by @patrickvonplaten

Reply:
pushed.

Reply:
Once the PR template is complete and everybody is happy I will sync with the Issue template

Reply:
I'm good @patrickvonplaten! I don't have time to follow, but I don't really mind being pinged, no big deal

Reply:
oh, @TevenLeScao - I already removed you - revert?

Reply:
It's fine, keep it that way, no worries